internal/infrastructure/mongo/mongodoc/user.go

```diff
@@ -86,6 +103,7 @@ func (d *UserDocument) Model() (*user1.User, error) {
 		LangFrom(d.Lang).
 		Password("", d.Password).
 		Theme(user.Theme(d.Theme)).
+		PasswordReset(d.PasswordReset.Token, d.PasswordReset.IsUsed, d.PasswordReset.ExpiresAt, d.PasswordReset.CreatedAt).
```

Comment:
Ifd.PasswordReset is nil, PasswordReset field in User should be nil, otherwise the app will be panic